spec/support/service_spec_helper.rb

```diff
@@ -35,6 +35,14 @@ def topological_url(partial_path, api_version = "v2.0")
     "http://topology/api/topological-inventory/#{api_version}/#{partial_path}"
   end
 
+  def approval_url(partial_path, api_version = "v1.0")
+    "http://approval/api/approval/#{api_version}/#{partial_path}"
```

Comment:
@syncrou can we use http://localhost/api/approval/#{api_version/#{partial_path}
Also if the partial_path has a leading slash will this have 2 slashes
can we use URI.join?